domains/calder.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "da-coder-jr",
+        "email": "calderlovelace@gmail.com"
+    },
+    "record": {
+        "CNAME": "da-coder-jr.github.io"
```

could you a little more about yourself as it has to be developer related?
```suggestion
    "records": {
        "CNAME": "da-coder-jr.github.io"
```